### PR TITLE
🌐 [i18n-KO] Translated `token_classification.mdx` to Korean

### PR DESCRIPTION
**Pull request by 0525hhgus**
# What does this PR do?

Translated the `tasks/token_classification.mdx` file of the documentation to Korean.

Thank you in advance for your review! 😄 

Part of https://github.com/huggingface/transformers/issues/20179

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Team PseudoLab, may you please review this PR?

@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/tasks/token_classification.mdx [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+
+- `B-` 엔터티의 시작을 나타냅니다.
+- `I-` 토큰이 동일한 엔터티 내부에 포함되어 있음을 나타냅니다 (예를 들어 `State` 토큰은 `Empire State Building`와 같은 엔터티의 일부입니다.).
+- `0` 토큰이 어떤 엔터티에도 해당하지 않음을 나타냅니다.
+
+## 전처리[[preprocess]]
+
+<Youtube id="iY2AZYdZAr0"/>
+
+다음 단계는 `tokens` 필드를 전처리하기 위해 DistilBERT 토크나이저를 가져오는 것입니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:103]:
```suggestion
다음으로 `tokens` 필드를 전처리하기 위해 DistilBERT 토크나이저를 가져옵니다:
```
자연스러운 문장을 고민하여 제안 드립니다!

File: docs/source/ko/tasks/token_classification.mdx [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+
+```py
+>>> from transformers import AutoTokenizer
+
+>>> tokenizer = AutoTokenizer.from_pretrained("distilbert-base-uncased")
+```
+
+위의 예제 'tokens' 필드에서 보았듯이 입력은 이미 토큰화된 것처럼 보입니다. 그러나 실제로 입력은 아직 토큰화되지 않았으므로 단어를 하위 단어로 토큰화하려면 `is_split_into_words=True`를 설정해야 합니다. 예를 들어:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:111]:
```suggestion
위의 예제 `tokens` 필드를 보면 입력이 이미 토큰화된 것처럼 보입니다. 그러나 실제로 입력은 아직 토큰화되지 않았으므로 단어를 하위 단어로 토큰화하기 위해 `is_split_into_words=True`를 설정해야 합니다. 예를 들어: 
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+
+```py
+>>> example = wnut["train"][0]
+>>> tokenized_input = tokenizer(example["tokens"], is_split_into_words=True)
+>>> tokens = tokenizer.convert_ids_to_tokens(tokenized_input["input_ids"])
+>>> tokens
+['[CLS]', '@', 'paul', '##walk', 'it', "'", 's', 'the', 'view', 'from', 'where', 'i', "'", 'm', 'living', 'for', 'two', 'weeks', '.', 'empire', 'state', 'building', '=', 'es', '##b', '.', 'pretty', 'bad', 'storm', 'here', 'last', 'evening', '.', '[SEP]']
+```
+
+그러나, 이는 `[CLS]`과 `[SEP]`라는 특수 토큰을 추가하고 하위 단어 토큰화로 인해 입력과 레이블 간에 불일치가 발생합니다. 이제 하나의 레이블에 해당하는 단일 단어가 두 개의 하위 단어로 분할될 수 있습니다. 토큰과 레이블을 다음과 같이 재정렬해야 합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:121]:
```suggestion
그러나 이로 인해 `[CLS]`과 `[SEP]`라는 특수 토큰이 추가되고, 하위 단어 토큰화로 인해 입력과 레이블 간에 불일치가 발생합니다. 하나의 레이블에 해당하는 단일 단어는 이제 두 개의 하위 단어로 분할될 수 있습니다. 토큰과 레이블을 다음과 같이 재정렬해야 합니다:
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+
+1. [`word_ids`](https://huggingface.co/docs/transformers/main_classes/tokenizer#transformers.BatchEncoding.word_ids) 메소드로 모든 토큰을 해당 단어에 매핑합니다.
+2. 특수 토큰 `[CLS]`와 `[SEP]`에 -100 레이블을 할당하여 PyTorch 손실 함수에 무시되도록 합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:124]:
```suggestion
2. 특수 토큰 `[CLS]`와 `[SEP]`에 `-100` 레이블을 할당하여, PyTorch 손실 함수가 해당 토큰을 무시하도록 합니다.
```
혹은
2. 특수 토큰 `[CLS]`와 `[SEP]`에 `-100` 레이블을 할당하여 PyTorch 손실 함수에게 무시되도록 합니다.
같은 문장은 어떨까요?

File: docs/source/ko/tasks/token_classification.mdx [hunk 5]
```diff
@@ -0,0 +1,556 @@
...
+...             else:
+...                 label_ids.append(-100)
+...             previous_word_idx = word_idx
+...         labels.append(label_ids)
+
+...     tokenized_inputs["labels"] = labels
+...     return tokenized_inputs
+```
+
+전체 데이터셋에 전처리 함수를 적용하려면, 🤗 Datasets [`~datasets.Dataset.map`] 함수를 사용하세요. 데이터셋의 여러 요소를 한 번에 처리하기 위해 `batched=True`로 설정함으로써 데이터셋 `map`를 더 빠르게 처리할 수 있습니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:152]:
```suggestion
전체 데이터 세트에 전처리 함수를 적용하려면, 🤗 Datasets [`~datasets.Dataset.map`] 함수를 사용하세요. `batched=True`로 설정하여 데이터 세트의 여러 요소를 한 번에 처리하면 `map` 함수의 속도를 높일 수 있습니다:
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 6]
```diff
@@ -0,0 +1,556 @@
...
+>>> from transformers import DataCollatorForTokenClassification
+
+>>> data_collator = DataCollatorForTokenClassification(tokenizer=tokenizer, return_tensors="tf")
+```
+</tf>
+</frameworkcontent>
+
+## 평가[[evaluation]]
+
+훈련 중 모델의 성능을 평가하기 위해 메트릭을 포함하는 것이 유용합니다. 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하여 빠르게 평가 방법을 로드할 수 있습니다. 이 작업에서는 [seqeval](https://huggingface.co/spaces/evaluate-metric/seqeval) 메트릭을 가져옵니다. (메트릭을 가져오고 계산하는 방법에 대해서는 🤗 Evaluate [quick tour](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요). Seqeval은 실제로 정밀도, 재현률, F1 및 정확도와 같은 여러 점수를 산출합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:178]:
```suggestion
훈련 중 모델의 성능을 평가하기 위해 평가 지표를 포함하는 것이 유용합니다. 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하여 빠르게 평가 방법을 가져올 수 있습니다. 이 작업에서는 [seqeval](https://huggingface.co/spaces/evaluate-metric/seqeval) 평가 지표를 가져옵니다. (평가 지표를 가져오고 계산하는 방법에 대해서는 🤗 Evaluate [quick tour](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요). Seqeval은 실제로 정밀도, 재현률, F1 및 정확도와 같은 여러 점수를 산출합니다.
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 7]
```diff
@@ -0,0 +1,556 @@
...
+...     tokenized_inputs["labels"] = labels
+...     return tokenized_inputs
+```
+
+전체 데이터셋에 전처리 함수를 적용하려면, 🤗 Datasets [`~datasets.Dataset.map`] 함수를 사용하세요. 데이터셋의 여러 요소를 한 번에 처리하기 위해 `batched=True`로 설정함으로써 데이터셋 `map`를 더 빠르게 처리할 수 있습니다:
+```py
+>>> tokenized_wnut = wnut.map(tokenize_and_align_labels, batched=True)
+```
+
+이제 [`DataCollatorWithPadding`]를 사용하여 예제 배치를 만들어봅시다. 데이터셋 전체를 최대 길이로 패딩하는 대신, *동적 패딩*을 사용하여 배치에서 가장 긴 길이에 맞게 문장을 패딩하는 것이 효율적입니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:157]:
```suggestion
이제 [`DataCollatorWithPadding`]를 사용하여 예제 배치를 만들어봅시다. 데이터 세트 전체를 최대 길이로 패딩하는 대신, *동적 패딩*을 사용하여 배치에서 가장 긴 길이에 맞게 문장을 패딩하는 것이 효율적입니다.
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 8]
```diff
@@ -0,0 +1,556 @@
...
+...     "B-product": 11,
+...     "I-product": 12,
+... }
+```
+
+<frameworkcontent>
+<pt>
+<Tip>
+
+[`Trainer`]를 사용하여 모델을 파인 튜닝하는 방법에 익숙하지 않은 경우, [여기](../training#train-with-pytorch-trainer)의 기본 튜토리얼을 확인하세요!
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:259]:
```suggestion
[`Trainer`]를 사용하여 모델을 파인 튜닝하는 방법에 익숙하지 않은 경우, [여기](../training#train-with-pytorch-trainer)에서 기본 튜토리얼을 확인하세요!
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 9]
```diff
@@ -0,0 +1,556 @@
...
+
+</Tip>
+
+이제 모델을 훈련시킬 준비가 되었습니다! [`AutoModelForSequenceClassification`]로 DistilBERT를 가쳐오고 예상되는 레이블 수와 레이블 매핑을 지정하세요:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:263]:
```suggestion
이제 모델을 훈련시킬 준비가 되었습니다! [`AutoModelForSequenceClassification`]로 DistilBERT를 가져오고 예상되는 레이블 수와 레이블 매핑을 지정하세요:
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 10]
```diff
@@ -0,0 +1,556 @@
...
+>>> from transformers import AutoModelForTokenClassification, TrainingArguments, Trainer
+
+>>> model = AutoModelForTokenClassification.from_pretrained(
+...     "distilbert-base-uncased", num_labels=13, id2label=id2label, label2id=label2id
+... )
+```
+
+이제 세 단계만 거치면 끝입니다:
+
+1. [`TrainingArguments`]에서 하이퍼파라미터를 정의하세요. `output_dir`는 모델을 저장할 위치를 지정하는 유일한 파라미터입니다. 이 모델을 Hub에 업로드하기 위해 `push_to_hub=True`를 설정합니다. (모델을 업로드하기 위해 Hugging Face에 로그인해야합니다.) 각 에폭이 끝날 때마다, [`Trainer`]는 seqeval 점수를 평가하고 훈련 체크포인트를 저장합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:275]:
```suggestion
1. [`TrainingArguments`]에서 하이퍼파라미터를 정의하세요. `output_dir`는 모델을 저장할 위치를 지정하는 유일한 매개변수입니다. 이 모델을 Hub에 업로드하기 위해 `push_to_hub=True`를 설정합니다(모델을 업로드하기 위해 Hugging Face에 로그인해야합니다.) 각 에폭이 끝날 때마다, [`Trainer`]는 seqeval 점수를 평가하고 훈련 체크포인트를 저장합니다.
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 11]
```diff
@@ -0,0 +1,556 @@
...
+2. [`Trainer`]에 훈련 인수와 모델, 데이터셋, 토크나이저, 데이터 수집기 및 `compute_metrics` 함수를 전달하세요.
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:276]:
```suggestion
2. [`Trainer`]에 훈련 인수와 모델, 데이터 세트, 토크나이저, 데이터 콜레이터 및 `compute_metrics` 함수를 전달하세요.
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 12]
```diff
@@ -0,0 +1,556 @@
...
+3. [`~Trainer.train`]를 호출하여 모델은 파인 튜닝하세요.
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:277]:
```suggestion
3. [`~Trainer.train`]를 호출하여 모델을 파인 튜닝하세요.
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 13]
```diff
@@ -0,0 +1,556 @@
...
+>>> num_train_steps = (len(tokenized_wnut["train"]) // batch_size) * num_train_epochs
+>>> optimizer, lr_schedule = create_optimizer(
+...     init_lr=2e-5,
+...     num_train_steps=num_train_steps,
+...     weight_decay_rate=0.01,
+...     num_warmup_steps=0,
+... )
+```
+
+그런 다음 [`TFAutoModelForSequenceClassification`]을 사용하여 DistilBERT를 로드하고, 예상되는 레이블 수와 레이블 매핑을 로드할 수 있습니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:334]:
```suggestion
그런 다음 [`TFAutoModelForSequenceClassification`]을 사용하여 DistilBERT를 가져오고, 예상되는 레이블 수와 레이블 매핑을 지정합니다:
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 14]
```diff
@@ -0,0 +1,556 @@
...
+... )
+```
+
+그런 다음 콜백을 함께 묶습니다:
+
+```py
+>>> callbacks = [metric_callback, push_to_hub_callback]
+```
+
+드디어, 모델 훈련을 시작할 준비가 되었습니다! [`fit`](https://keras.io/api/models/model_training_apis/#fit-method)에 훈련 데이터셋, 검증 데이터셋, 에폭의 수 및 콜백을 전달하여 파인 튜닝합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/token_classification.mdx:397]:
```suggestion
드디어, 모델 훈련을 시작할 준비가 되었습니다! [`fit`](https://keras.io/api/models/model_training_apis/#fit-method)에 훈련 데이터 세트, 검증 데이터 세트, 에폭의 수 및 콜백을 전달하여 파인 튜닝합니다:
```

**Review by sim-so — CHANGES_REQUESTED** [review]
정말 많은 내용이 들어 있는 문서였지만 덕분에 금방 읽을 수 있었습니다! 😄 
`Named Entity Recognition` -> `개체명 인식`, `dataset`->`데이터셋` 두 가지를 포함하여 몇 가지 수정 사항을 제안 드립니다. 잘 부탁 드리겠습니다!

File: docs/source/ko/tasks/token_classification.mdx [hunk 15]
```diff
@@ -0,0 +1,556 @@
...
+
+```py
+>>> model.fit(x=tf_train_set, validation_data=tf_validation_set, epochs=3, callbacks=callbacks)
+```
+
+훈련이 완료되면, 모델이 자동으로 Hub에 업로드되어 누구나 사용할 수 있습니다!
```
Comment (0525hhgus) [inline docs/source/ko/tasks/token_classification.mdx:403]:
```suggestion
훈련이 완료되면, 모델이 자동으로 허브에 업로드되어 누구나 사용할 수 있습니다!
```

File: docs/source/ko/tasks/token_classification.mdx [hunk 16]
```diff
@@ -0,0 +1,556 @@
...
+
+다음으로 `tokens` 필드를 전처리하기 위해 DistilBERT 토크나이저를 가져옵니다:
+
+```py
+>>> from transformers import AutoTokenizer
+
+>>> tokenizer = AutoTokenizer.from_pretrained("distilbert-base-uncased")
+```
+
+위의 예제 `tokens` 필드를 보면 입력이 이미 토큰화된 것처럼 보입니다. 그러나 실제로 입력은 아직 토큰화되지 않았으므로 단어를 하위 단어로 토큰화하기 위해 `is_split_into_words=True`를 설정해야 합니다. 예를 들어:
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/token_classification.mdx:111]: 예를들어:
를 예제로 확인합니다.
로 작성하면 좀 더 편하게 이해할 수 있을 것 같습니다.

Comment (0525hhgus) [reply docs/source/ko/tasks/token_classification.mdx]: 리뷰 감사합니다 👍 완성형으로 끝나서 가독성이 더 좋아졌습니다 😄

File: docs/source/ko/tasks/token_classification.mdx [hunk 17]
```diff
@@ -0,0 +1,556 @@
...
+>>> from transformers import DataCollatorForTokenClassification
+
+>>> data_collator = DataCollatorForTokenClassification(tokenizer=tokenizer, return_tensors="tf")
+```
+</tf>
+</frameworkcontent>
+
+## 평가[[evaluation]]
+
+훈련 중 모델의 성능을 평가하기 위해 평가 지표를 포함하는 것이 유용합니다. 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하여 빠르게 평가 방법을 가져올 수 있습니다. 이 작업에서는 [seqeval](https://huggingface.co/spaces/evaluate-metric/seqeval) 평가 지표를 가져옵니다. (평가 지표를 가져오고 계산하는 방법에 대해서는 🤗 Evaluate [quick tour](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요). Seqeval은 실제로 정밀도, 재현률, F1 및 정확도와 같은 여러 점수를 산출합니다.
```
[the remark this answers is not in the document]
Comment (0525hhgus) [reply docs/source/ko/tasks/token_classification.mdx:178]: 꼼꼼한 리뷰 감사합니다! 커밋에 반영했습니다 😄

**Comment by 0525hhgus** [comment]
May you please review this PR? 😄 
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**